src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.wasm.cs

```diff
@@ -2,12 +2,13 @@
 
 namespace Windows.ApplicationModel.DataTransfer
 {
-	public partial class Clipboard
+	public static partial class Clipboard
 	{
 		public static void SetContent(DataPackage content)
 		{
 			var text = WebAssemblyRuntime.EscapeJs(content.Text);
```

Comment:
Normally, yes and we can even have multiple content type available (eg: `Text`, `System.String`, `XML Spreadsheet`, `HTML Format` being available when copying from excel). However, given how `DataPackage` is being implemented, only text content can be set (and uri content which is implemented as text for `WAsm`).

You can passing `null` (both `content` or `content.text`) here, since there is no guard for it.

Reply:
This feature were already supported and used in the playground...

https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.Shared/Samples/Playground.xaml.cs#L425